docs/control-structures.rst

```diff
@@ -18,12 +18,17 @@ Functions are executable units of code within a contract. Functions may only be
 
 Functions may be called internally or externally depending on their :ref:`visibility <function-visibility>`. Functions may accept input arguments and return variables in order to pass values between them.
 
-.. _function-visibility:
-
 Visibility
 ----------
 
-All functions must include exactly one visibility decorator.
+.. _function-visibility:
+
+You can optionally declare a function's visibility by using a :ref:`decorator <function-decorators>`. There are three visibility levels in vyper:
+
+    * **External**: exposed in the selector table, can be called by an external call into this contract
+    * **Internal**: can be invoked only from within this contract. not available to external callers
+    * **Deploy**: constructor code. This is code which is invoked once in the lifetime of a contract, upon its deploy. It is not available at runtime to either external callers or internal call invocations. Currently, only the :ref:`__init__() function <init-function>` may be marked as ``@deploy``.
```

Comment:
```suggestion
    * ``@deploy``: constructor code. This is code which is invoked once in the lifetime of a contract, upon its deploy. It is not available at runtime to either external callers or internal call invocations. Currently, only the :ref:`__init__() function <init-function>` must be marked as ``@deploy``.
```

Reply:
you can't mark any other functions as `@deploy`

Reply:
Oh, I re-read the sentence, and yeah ignore the suggestion _must_